frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -49,7 +49,7 @@ export const isPodSchedulable = (pod: PodKind) => {
 const isPodReady = (pod: PodKind): boolean =>
   pod?.status?.phase === 'Running' && pod?.status?.containerStatuses?.every((s) => s.ready);
 
-export const findVMIPod = (
+export const getPodsForVMI = (
```

Comment:
Do we really want a util to get all pods to display in the sidebar instead of just the one pod as displayed in their details page?